blueprints/networking/glb-hybrid-neg-internal/README.md

```diff
@@ -0,0 +1,103 @@
+# XGLB and multi-regional daisy-chaining through hybrid NEGs
```

Comment:
Can you also add it to all the top-level READMEs in the parent folders?

Reply:
done